locale/en.po

```diff
@@ -1960,6 +1960,26 @@ msgstr "\"{0}\" - {1}"
 msgid "ART_BY"
 msgstr "Art by {0}"
 
+#: ../src/general/BehaviourDictionary.cs:102
+msgid "BEHAVIOUR_AGGRESSION"
+msgstr "Aggression"
```

Comment:
Don't these translations already exist for the GUI? This new feature should use the existing translation keys.

Reply:
Oh it should.

Reply:
No they don't. The current ones are like

```gettext
#: ../src/microbe_stage/editor/BehaviourEditorSubComponent.tscn:141
msgid "OPPORTUNISTIC"
msgstr "Opportunistic"
```

Reply:
I guess then we need these new translations.